### ISSUE TITLE
model.eval() with ensemble of CNN is not extended to the sub models 

### ISSUE THREAD

**Issue by ZSoumia** [pytorch/pytorch#23309]
## 🚀 Feature
extending the .eval() to  the sub models of an ensemble of neural networks .

## Motivation
I trained an ensemble of CNN models ( Squeezenet1_1 trained on different training sets ) and saved the checkpoint , when loading the checkpoint for inference *Even if I called the .eval() on the ensemble model*  I had inconsistent results because the sub models were not set to the evaluation mode . I think is because the models were loaded as a list of models inside of the ensemble model aggregating them. 

## Pitch
Defining a structure to aggregate different checkpoints into one model ( instead of a list ) or defining a way we can extend the .eval() effect on the sub models inside of an ensemble model .

**Comment by SsnL** [comment]
Use ModuleList.
